gprofiler/utils/perf.py

```diff
@@ -3,14 +3,71 @@
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
 
-from gprofiler.exceptions import CalledProcessError
+from enum import Enum
+from pathlib import Path
+from threading import Event
+
+from gprofiler.exceptions import CalledProcessError, PerfNoSupportedEvent
 from gprofiler.log import get_logger_adapter
 from gprofiler.utils import run_process
-from gprofiler.utils.perf_process import perf_path
+from gprofiler.utils.perf_process import PerfProcess, perf_path
 
 logger = get_logger_adapter(__name__)
 
 
+class SUPPORTED_PERF_EVENTS(Enum):
+    PERF_DEFAULT = "default"
+    PERF_SW_CPU_CLOCK = "cpu-clock"
+    PERF_SW_TASK_CLOCK = "task-clock"
+
+    def perf_extra_args(self) -> list:
```

Comment:
```suggestion
    def perf_extra_args(self) -> List[str]:
```

Reply:
it can return empty list tho when `self == SUPPORTED_PERF_EVENTS.PERF_DEFAULT`
so `List`

Reply:
b789ea97cedfff3cc34c3a60784c214a875e083a